### PR TITLE
Add move request to job factory

### PR DESCRIPTION
**Pull request by ssyssy**
### What changes are proposed in this pull request?

Add move job request

### Why are the changes needed?

Need for distMv command.

### Does this PR introduce any user facing changes?

na

### CODE REVIEW

File: dora/core/server/master/src/main/java/alluxio/master/job/MoveJobFactory.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    boolean checkContent = options.hasCheckContent() && options.getCheckContent();
+    UnderFileSystem ufs = mFs.getUfsManager().getOrAdd(new AlluxioURI(src),
+        UnderFileSystemConfiguration.defaults(Configuration.global()));
+    Iterable<FileInfo> fileIterator = new UfsFileIterable(ufs, src, Optional
+        .ofNullable(AuthenticatedClientUser.getOrNull())
+        .map(User::getName), partialListing, FileInfo::isCompleted);
+    Optional<String> user = Optional
+        .ofNullable(AuthenticatedClientUser.getOrNull())
+        .map(User::getName);
+    return new CopyJob(src, mRequest.getDst(), overwrite, user, UUID.randomUUID().toString(),
```
Comment (jja725) [inline dora/core/server/master/src/main/java/alluxio/master/job/MoveJobFactory.java:58]: Copy??

**Review by jja725 — CHANGES_REQUESTED** [review]

**Review by jja725 — APPROVED** [review]
LGTM

**Comment by ssyssy** [comment]
alluxio-bot, merge this please

**Comment by ssyssy** [comment]
alluxio-bot, merge this please

**Pull request merged**